### ISSUE TITLE
Adjust TF Tuning to Incorporate Different Loss Functions without Predefinition

### ISSUE THREAD

**Issue by rsethi21** [lellew/Peptide_Folding_Classifier#50]
https://adriangb.com/scikeras/stable/generated/scikeras.wrappers.KerasClassifier.html

**Comment by rsethi21** [comment]
Issue is that the keras classifier has default parameters for loss and optimizer, remove that part from the functions
